kafka-connect-s3/src/main/java/io/confluent/connect/s3/format/avro/AvroRecordWriterProvider.java

```diff
@@ -102,7 +102,7 @@ public void commit() {
           s3out.commit();
           writer.close();
         } catch (IOException e) {
-          throw new RetriableException(e);
+          throwConnectException(e);
         }
       }
 
```

Comment:
@lion-simba and @cjolivier01 any reasons why the below `close()` method directly throws ConnectException instead of `throwConnectException(e)`, should this also be retriable?

Reply:
appears fixed

Reply:
@cjolivier01 please check line 114 in this class as shown in screenshot I do not see it is fixed :).
![image](https://user-images.githubusercontent.com/29556518/152866834-03088438-f308-4088-92a9-64353df6a8b6.png)

Reply:
@kpatelatwork,
I didn't change it in first place, because there is no place in code, which can handle RetriableException thrown from here. But it shouldn't harm either.